guides/source/active_record_querying.md

```diff
@@ -2187,6 +2187,74 @@ NOTE: Note that if a query matches multiple records, `find_by` will
 fetch only the first one and ignore the others (see the `LIMIT 1`
 statement above).
 
+### Using Common Table Expression (CTE)
+
+The [`with`][] method is a convenient way to access [Common Table Expressions (CTE)](https://en.wikipedia.org/wiki/Hierarchical_and_recursive_queries_in_SQL#Common_table_expression) `WITH` syntax, which can define named temporary query results that are referable within a single SQL query.
```

Comment:
The formatting of the CTE abbreviation/link with the `WITH` syntax is a bit hard to read:
<img width="447" alt="image" src="https://github.com/user-attachments/assets/516f80bf-635c-44c0-8aed-fd7a90348a54">

Maybe first explain what CTEs are, before explaining the `with` method?
We probably don't need explain the abbreviation twice ("Common Table Expressions (CTE)")? (Maybe it's better to do it here instead of the header?)

Maybe something like:
```suggestion
[Common Table Expressions](https://en.wikipedia.org/wiki/Hierarchical_and_recursive_queries_in_SQL#Common_table_expression) are named temporary query results that are referable within a single SQL query. The [`with`][] method is a convenient way to access  CTEs `WITH` syntax.
```

Reply:
Revised the paragraph!